README.md

```diff
@@ -60,6 +60,18 @@ myApp.run(function(featureFlags, $http) {
 });
 ```
 
+### Setting flag data on config phase (≥ v1.0.1)
```

Comment:
I was thinking this pull request is a bit bigger than a patch and was going to update it to v1.1.0

Can you update these v1.0.1 references to v1.1.0 please